package.json

```diff
@@ -88,20 +81,24 @@
     "fork-ts-checker-webpack-plugin": "^6.3.3",
     "husky": "^3.0.4",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^28.1.2",
-    "jest-cli": "^28.1.2",
-    "jest-environment-jsdom": "^28.1.2",
-    "jest-when": "3.5.1",
+    "jest": "^29.5.0",
+    "jest-cli": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "jest-when": "^3.5.2",
     "openmrs": "next",
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1",
     "react": "^18.2.0",
     "react-i18next": "^11.18.1",
     "rxjs": "^6.6.3",
+    "swc-loader": "^0.2.3",
     "typescript": "^4.0.3",
-    "webpack": "^5.37.0",
-    "webpack-cli": "^4.7.0",
-    "webpack-dev-server": "^4.2.1"
+    "webpack": "^5.76.2",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.13.1"
+  },
+  "resolutions": {
+    "@carbon/react": "1.13.0"
```

Comment:
What happens if we get rid of the resolutions?

Reply:
There's another package pulling in a different version of `@carbon/react` specifically `v1.25.x` which created issues so this just locks the preference

Reply:
Asking because we have to keep in mind that defining resolutions for core packages like "@carbon/react" should be done apprehensively